upgrade/upgrade-scripts/upgrade

```diff
@@ -20,8 +20,6 @@
 IMAGE_PATH=$(get_image_path)
 [[ -n "$IMAGE_PATH" ]] || die "failed to determine image path"
 
-IMAGE_VERSION=$(get_image_version)
-[[ -n "$IMAGE_VERSION" ]] || die "failed to determine image version"
 
```

Comment:
it looks like shfmt only wants a single empty line here; see [also](https://github.com/delphix/appliance-build/pull/543/checks?check_run_id=2496161744).